helix-core/src/test/java/org/apache/helix/manager/zk/TestRawZkClient.java

```diff
@@ -819,4 +820,45 @@ public void testAsyncWriteOperations() {
       zkClient.delete("/tmp/asyncOversize");
     }
   }
+
+  @Test(expectedExceptions = ZkMarshallingError.class,
```

Comment:
Thanks for adding this test! One favor to ask - we are actually in the process of moving this class to zookeeper-api (https://github.com/apache/helix/pull/1070).

Do you think we could hold off on this change until that is committed first to avoid merge conflicts?